src/points/slice.ts

```diff
@@ -1,10 +1,10 @@
 import { PayloadAction, createSlice } from '@reduxjs/toolkit'
-import { ClaimHistory, PointsActivityId } from 'src/points/types'
+import { RawClaimHistory, PointsActivityId, isClaimActivityId } from 'src/points/types'
 import { REHYDRATE, RehydrateAction, getRehydratePayload } from 'src/redux/persist-helper'
 
 interface GetPointsHistorySucceededAction {
   appendHistory: boolean
-  newPointsHistory: ClaimHistory[]
+  newPointsHistory: RawClaimHistory[]
```

Comment:
small nit: i personally don't like prefixing names with `raw` because it doesn't really help us to understand what it means (aside from the one scenario of being used inside a function that massages data, but even then there are more descriptive names we could use). perhaps we can stick with `ClaimHistory` since that describes the data that we store, and the return type of the selector can be something like `ClaimHistoryCardItem` or `ClaimHistoryItemWithTimestamp`? it would make it clearer what the type is for (i.e. display purposes)

Reply:
yeah i didn't love this name either -- updated to use the original meaning of `ClaimHistory`, and the timestamped version now uses `ClaimHistoryCardItem`